nativescript-angular/platform-common.ts

```diff
@@ -50,11 +50,18 @@ if ((<any>global).___TS_UNUSED) {
         return InjectionToken;
     })();
 }
+
+export interface HmrOptions {
+    moduleTypeFactory?: () => Type<any> | NgModuleFactory<any>;
```

Comment:
Maybe add a comment why we need that to be a factory function instead of the module itself?

Reply:
Done